src/test/java/uk/gov/companieshouse/api/accounts/service/impl/StatementServiceTest.java

```diff
@@ -1,6 +1,7 @@
 package uk.gov.companieshouse.api.accounts.service.impl;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
```

Comment:
Can we add a test or two to this class for the case where `ProfitAndLossService#find` returns a `NOT_FOUND` status?